.gitignore

```diff
@@ -21,3 +21,4 @@ benchmark.txt
 npm_licenses.tar.bz2
 /web/ui/static/react
 /web/ui/assets_vfsdata.go
+.DS_Store
```

Comment:
Do we need this?

Reply:
No, our policy is not include any files not produced/related to the repo itself. This is something for your global gitconfig.

Reply:
I use MacOs and .DS_Store is a file that stores custom attributes of its containing folder. It is created by my file finder everytime. Hence, I put it in gitignore. I can remove it if there is a workaround to not committing it everytime i do `git add . ` ( Lemme ask MacUsers )

Reply:
Adding it to my global gitignore works . Removing it from here.